File: app/src/main/java/com/fphoenixcorneae/viewpagertransformer/DefaultTransformer.java
package com.fphoenixcorneae.viewpagertransformer;

import android.view.View;

/**
 * Created on 2017/6/30.
 */

public class DefaultTransformer extends BaseTransformer {
    @Override
    protected void onTransform(View page, float position) {

    }

    @Override
    protected boolean isPagingEnabled() {
        return true;
    }
}

File: view-pager-transformer/src/main/java/com/fphoenixcorneae/view_pager_transformer/DefaultTransformer.kt
package com.fphoenixcorneae.view_pager_transformer

import android.view.View

/**
 * @desc 默认转换效果
 * @date 2017-06-30
 */
class DefaultTransformer : AbstractBaseTransformer() {
    override fun onTransform(page: View, position: Float) {}
    override val isPagingEnabled: Boolean = true
}